apps/app.near/widget/Components/Profile/PokeButton.jsx

```diff
@@ -0,0 +1,44 @@
+const { Button } = VM.require("app.near/widget/Components") || {
+  Button: () => <></>,
+};
+
+if (
+  !props.accountId ||
+  !context.accountId ||
+  context.accountId === props.accountId
+) {
+  return "";
+}
+
+const data = {
```

Comment:
Could you move this to a "poke" function in an sdk -- let's put it at "sdk/social"?

poke(accountId) {
 Social.set(....)
}